src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,101 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final double DEFAULT_COEFFICIENT = 1.5;
+    private static final String INVALID_INDEX_MSG = "The index passed to the method is invalid";
+    private static final int DEFAULT_LENGTH = 10;
+    private int arrayLength = DEFAULT_LENGTH;
```

Comment:
Okey, but why do you need to do it? 
If you need to get an array length, you can just use the `length` property of array
````
array.length
````